Sources/Extensions/UIWindow+Deboogger.swift

```diff
@@ -21,6 +34,8 @@ extension UIWindow {
         recognizer.numberOfTouchesRequired = Constants.numberOfTouches
```

Comment:
It shouldn't use `Constants` to configure gesture parameters - if window is created after `setup(gesture)` call in Deboogger, it'll use incorrect gesture configuration.